mods/ts/weapons/energyweapons.yaml

```diff
@@ -124,7 +124,6 @@ Proton:
 		ImpactSound: ssplash3.aud
 		ValidImpactTypes: Water
 
-# Obelisk laser
```

Comment:
This comment should be kept or `LaserFire` should be renamed to `ObeliskLaser`[`Fire`].